spec/client/ics-006-solo-machine-client/README.md

```diff
@@ -70,22 +70,37 @@ The `Height` of a solo machine is just a `uint64`, with the usual comparison ope
 
 ```typescript
 interface Header {
-  sequence: uint64
```

Comment:
Why is the sequence removed? see https://github.com/cosmos/ibc-go/blob/cd913b8fab4017e7dd7953904faba23aaba13837/modules/light-clients/06-solomachine/solomachine.pb.go#L122